src/Command/CreateDatabaseDoctrineCommand.php

```diff
@@ -67,7 +67,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         unset($params['dbname'], $params['path'], $params['url']);
 
         if ($connection->getDatabasePlatform() instanceof PostgreSQLPlatform) {
-            /** @psalm-suppress InvalidArrayOffset It's still available in DBAL 3.x that we need to support */
+            /**
+             * for DBAL < 4
+             *
+             * @phpstan-ignore nullCoalesce.offset
+             */
```

Comment:
you should be able to add comments like this
```suggestion
            /** @phpstan-ignore nullCoalesce.offset (needed for DBAL < 4) */
```

pls apply to all of these instances